sdcm/cluster.py

```diff
@@ -1894,6 +1896,15 @@ def stop_scylla_manager_log_capture(self, timeout=10):
         self._scylla_manager_journal_thread.join(timeout)
         self._scylla_manager_journal_thread = None
 
+    def config_scylla_manager_yaml(self, segments_per_repair=None):
```

Comment:
The solution is error-prone. Need to bring the yaml, parse it, change the values dumps and upload.